learning_journal/views.py

```diff
@@ -1,21 +1,77 @@
 from pyramid.response import Response
 from pyramid.view import view_config
+from pyramid.httpexceptions import HTTPFound
 
-from sqlalchemy.exc import DBAPIError
+from sqlalchemy.exc import DBAPIError, SQLAlchemyError
+from sqlalchemy import desc
 
 from .models import (
     DBSession,
     Entry,
     )
 
+from wtforms import Form, StringField, TextAreaField, validators
+import markdown
 
-@view_config(route_name='home', renderer='templates/mytemplate.pt')
-def my_view(request):
-    try:
-        one = DBSession.query(Entry).filter(Entry.name == 'one').first()
-    except DBAPIError:
-        return Response(conn_err_msg, content_type='text/plain', status_int=500)
-    return {'one': one, 'project': 'learning-journal'}
+class EntryForm(Form):
+    title = StringField(u'Title', [validators.required(),
+                        validators.length(max=128)])
+    text = TextAreaField(u'Entry', [validators.required()])
+
+
+@view_config(route_name='list', renderer='templates/list_template.jinja2')
+def list_view(request):
+    entries = DBSession.query(Entry).order_by(desc(Entry.created))
+    return {'entries': entries, 'title': 'Learning Journal'}
```

Comment:
I'm curious why you are sending the title of the learning journal back from this view.  Seems an odd choice.